static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -304,7 +304,6 @@ function TaskGroup({
                 task={task}
                 hidePanel={hidePanel}
                 showWaitingIndicator={taskKeyForWaitingIndicator === task.task}
-                status={task.status}
```

Comment:
when we pass the status here, if it’s 'complete' but not yet 'seen,' the task is displayed with the style of a complete task but does not appear in the completed list. To fix this issue, it’s better to omit the status in this context

Reply:
Shouldn't we instead fix the check if the group by completed/non completed tasks in `groupTasksByCompletion`?

https://github.com/getsentry/sentry/blob/2f6de175778330ac30ef6abc673ca56939349a3c/static/app/components/onboardingWizard/newSidebar.tsx#L54

We check there if the task is seen and completed.